---
layout: archive
title: "Publications"
permalink: /research/
author_profile: true
---

<head>
<style>
.abstract {
  display: none;
  background-color: lightgray;
  padding: 20px;
  width: 70%;
}
.toggle-button {
  display: block;
  width: 70%;
  padding: 10px 10px 10px;
  background-color: lightgray;
  border: none;
  cursor: pointer;
  text-align: left;
}
</style>
</head>


<body>

  <br>

  <h3>Planning for Gold: Sample Splitting for Valid Powerful Design of Observational Studies</h3>
  <p><b>William Bekerman</b>, Abhinandan Dalal, Carlo del Ninno, Dylan S. Small<br>
    <a href='https://arxiv.org/abs/2406.00866' id='link'>[link]</a>
    <a href='https://github.com/WillBekerman/planning-for-gold' id='link'>[code]</a>
  </p>

  <button class='toggle-button' onclick="showAbstract('abstract1')">Abstract</button>

  <p class='abstract' id="abstract1">Observational studies are valuable tools for inferring causal effects in the absence of controlled experiments. However, these studies may be biased due to the presence of some relevant, unmeasured set of covariates. The design of an observational study has a prominent effect on its sensitivity to hidden biases, and the best design may not be apparent without examining the data. One approach to facilitate a data-inspired design is to split the sample into a planning sample for choosing the design and an analysis sample for making inferences. We devise a powerful and flexible method for selecting outcomes in the planning sample when an unknown number of outcomes are affected by the treatment. We investigate the theoretical properties of our method and conduct extensive simulations that demonstrate pronounced benefits, especially at higher levels of allowance for unmeasured confounding. Finally, we demonstrate our method in an observational study of the multi-dimensional impacts of a devastating flood in Bangladesh.</p>

  <br>

  <h3>RZiMM-scRNA: A regularized zero-inflated mixture model framework for single-cell RNA-seq data</h3>
  <p>Xinlei Mi, <b>William Bekerman</b>, Anil K Rustgi, Peter A Sims, Peter D Canoll, Jianhua Hu<br>
    <a href='https://projecteuclid.org/journals/annals-of-applied-statistics/volume-18/issue-1/RZiMM-scRNA--A-regularized-zero-inflated-mixture-model-framework/10.1214/23-AOAS1761.short' id='link'>[link]</a>
    <a href='https://github.com/SkadiEye/RZiMM' id='link'>[code]</a>
  </p>

  <button class='toggle-button' onclick="showAbstract('abstract2')">Abstract</button>

  <p class='abstract' id="abstract2">Applications of single-cell RNA sequencing in various biomedical research areas have been blooming. This new technology provides unprecedented opportunities to study disease heterogeneity at the cellular level. However, unique characteristics of scRNA-seq data, including large dimensionality, high dropout rates, and possibly batch effects, bring great difficulty into the analysis of such data. Not appropriately addressing these issues obstructs true scientific discovery. Herein we propose a unified Regularized Zero-inflated Mixture Model framework, designed for scRNA-seq data (RZiMM-scRNA), to simultaneously detect cell subgroups and identify gene differential expression based on a developed importance score, accounting for both dropouts and batch effects. We conduct extensive simulation studies in which we evaluate the performance of RZiMM-scRNA and compare it with several popular methods, including Seurat, SC3, K-means, and hierarchical clustering. Simulation results show that RZiMM-scRNA demonstrates superior clustering performance and enhanced biomarker detection accuracy, compared to alternative methods, especially when cell subgroups are less distinct, verifying the robustness of our method.

Our empirical investigations focus on two brain tumor studies dealing with astrocytoma of various grades, including the most malignant of all brain tumors, glioblastoma multiforme (GBM). Our goal is to delineate cell heterogeneity and identify driving biomarkers associated with these tumors. Notably, RZiMM-scNRA successfully identifies a small group of oligodendrocyte cells, which has drawn much attention in biomedical literature on brain cancers. In addition, our method discovers several new biomarkers which are not discussed in the original studies, including PLP1, BCAN, and PTPRZ1—all associated with the development and malignant growth of glioma—as well as CAMK2B, which is downregulated in glioma and GBM and implicated in neurodevelopment, brain function, learning and memory processes.</p>

<script>
function showAbstract(abstractId) {
  var paragraph = document.getElementById(abstractId);
  if (paragraph.style.display === "none") {
    paragraph.style.display = "block";
  } else {
    paragraph.style.display = "none";
  }
}
</script>

</body>